inference/enterprise/helm-chart/roboflow-inference-server/values-cpu.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for roboflow-inference-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+# Type of environment - can be cpu or gpu
+computeType: cpu
+
+image:
+  repository: roboflow/roboflow-inference-server-cpu
+  tag: 0.9.7
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Environment variables to set
+# More information: 
+# https://inference.roboflow.com/quickstart/docker_configuration_options/
+envMap:
```

Comment:
here is the thing with `NUM_WORKERS`:
* the more workers the more parallel instances of the same model can be initialised (memory consumption grows)
* the more CPU request/resource assigned - the better to have more workers
* it would be healthy to limit per-worker CPU usage (via limiting runtime threads for models - not yet implemented) - if that's not the case - in practice under heavy load the service may experience throttling enforced by k8s

Reply:
there is a benefit of multiple workers at single pod - since this API is compute heavy, single worker will be busy for the whole time of request processing (even if part of processing does not consume whole resources) - so "workers over-provisionong" may reduce values of high percentiles of latency

Reply:
In my testing back in Jul/Aug I found NUM_WORKER > 1 was causing some GPU issues (cant recall what exactly, but it lead to server crashing). But for CPU its 100% beneficial to have higher NUM_WORKERs.